WebApp/src/uk/ac/exeter/QuinCe/web/files/UploadedDataFile.java

```diff
@@ -134,6 +134,13 @@ public String getMessageClass() {
 		return messageClass;
 	}
 
+	public String getMissingRunTypeClass() {
+		if (null != dataFile && dataFile.getMissingRunTypes().size() > 0) {
+			return "show";
```

Comment:
This should be `shown`

Reply:
Incidentally, I think there's a few places where we reference CSS classes in the code. Should we have them statically defined somewhere? If you agree we can make a Medium Term issue for it.